src/sinks/kafka/config.rs

```diff
@@ -159,79 +157,73 @@ impl KafkaSinkConfig {
-                    debug!(
-                        librdkafka_option = key,
-                        batch_option = "max_events",
-                        value,
-                        "Applying batch option as librdkafka option."
-                    );
-                    client_config.set(key, &value.to_string());
                 }
-                if let Some(value) = self.batch.max_bytes {
-                    // Maximum size (in bytes) of all messages batched in one MessageSet, including protocol
-                    // framing overhead. This limit is applied after the first message has been added to the
-                    // batch, regardless of the first message's size, this is to ensure that messages that
-                    // exceed batch.size are produced. The total MessageSet size is also limited by
-                    // batch.num.messages and message.max.bytes.
-                    // Type: integer
-                    let key = "batch.size";
-                    if let Some(val) = self.librdkafka_options.get(key) {
-                        return Err(format!("Batching setting `batch.max_bytes` sets `librdkafka_options.{}={}`.\
+                debug!(
+                    librdkafka_option = key,
+                    batch_option = "max_events",
+                    value,
+                    "Applying batch option as librdkafka option."
+                );
+                client_config.set(key, &value.to_string());
+            }
+            if let Some(value) = self.batch.max_bytes {
+                // Maximum size (in bytes) of all messages batched in one MessageSet, including protocol
+                // framing overhead. This limit is applied after the first message has been added to the
+                // batch, regardless of the first message's size, this is to ensure that messages that
+                // exceed batch.size are produced. The total MessageSet size is also limited by
+                // batch.num.messages and message.max.bytes.
+                // Type: integer
+                let key = "batch.size";
+                if let Some(val) = self.librdkafka_options.get(key) {
+                    return Err(format!("Batching setting `batch.max_bytes` sets `librdkafka_options.{}={}`.\
                                         The config already sets this as `librdkafka_options.batch.size={}`.\
                                         Please delete one.", key, value, val).into());
-                    }
-                    debug!(
-                        librdkafka_option = key,
-                        batch_option = "max_bytes",
-                        value,
-                        "Applying batch option as librdkafka option."
-                    );
-                    client_config.set(key, &value.to_string());
                 }
-            }
-
-            KafkaRole::Consumer => {
-                client_config.set("queued.min.messages", QUEUED_MIN_MESSAGES.to_string());
```

Comment:
This is unnecessary as it is the default value ([ref](https://github.com/confluentinc/librdkafka/blob/master/CONFIGURATION.md)).